src/test/java/com/yahoo/bullet/parsing/QueryUtils.java

```diff
@@ -253,10 +268,10 @@ public static String makeStringAttributesAggregation(Integer size, Aggregation.T
                                                          Map<String, String> attributes,
                                                          Pair<String, String>... fields) {
         return "{" +
-                 "'type' : '" + getTypeFor(operation) + "', " +
-                 "'fields' : " + makeGroupFields(fields) + ", " +
-                 "'size' : " + size + ", " +
-                 "'attributes' : " + makeMap(attributes) +
+                "'type' : '" + getTypeFor(operation) + "', " +
```

Comment:
These were being lined up with the second ```"```. Let's keep it that way for everything?

Reply:
done.